### PR TITLE
Adding personal code screen and authenticationMiddleware

### PR DESCRIPTION
**Pull request by abhiahirv1**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: test/src/controllers/personsNameController.test.ts [hunk 1]
```diff
@@ -1,22 +1,16 @@
...
-    middleName: "",
-    lastName: "Doe"
-};
-
 describe("GET" + PERSONS_NAME, () => {
     it("should return status 200", async () => {
         const res = await router.get(BASE_URL + PERSONS_NAME);
         expect(res.status).toBe(200);
         expect(mocks.mockSessionMiddleware).toHaveBeenCalled();
+        expect(mocks.mockAuthenticationMiddleware).toHaveBeenCalled();
```
Comment (ttingle-ch) [inline test/src/controllers/personsNameController.test.ts:13]: Can you add expect statement for page title

File: test/src/controllers/personsNameController.test.ts [hunk 2]
```diff
@@ -31,11 +25,18 @@ describe("POST" + PERSONS_NAME, () => {
...
     // Test for incorrect form details entered, will return 400.
     it("should return status 400 after incorrect data entered", async () => {
-        const res = await router.post(BASE_URL + PERSONS_NAME);
+        const sendData = {
+            "first-name": "",
+            "middle-names": "",
+            "last-name": ""
+        };
+        const res = await router.post(BASE_URL + PERSONS_NAME).send(sendData); ;
         expect(res.status).toBe(400);
```
Comment (ttingle-ch) [inline test/src/controllers/personsNameController.test.ts:40]: Can you add expect for error message and further tests for other error scenarios

File: src/types/pageURL.ts [hunk 3]
```diff
@@ -19,3 +15,9 @@ export const ACCOUNTS_SIGNOUT_PATH = `${ACCOUNT_URL}/signout`;
 export const HEALTHCHECK = "/healthcheck";
 
 export const ACCESSIBILITY_STATEMENT = "/accessibility-statement";
+
+export const PERSONS_NAME = "/what-is-the-persons-name";
+
+export const PERSONAL_CODE = "/companies-house-personal-code";
+
+export const WHAT_IS_EMAIL_ADDRESS = "/what-is-their-email-address";
```
Comment (sardhani-ch) [inline src/types/pageURL.ts:23]: to maintain consistency with persons name, can you change WHAT_IS_EMAIL_ADDRESS to EMAIL_ADDRESS please

File: test/src/controllers/personsNameController.test.ts (hunk 2 above)
Comment (abhiahirv1) [reply test/src/controllers/personsNameController.test.ts:50]: Done

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by swalke-ch — APPROVED** [review]

**Pull request merged**